### PR TITLE
allow user to set the thickness of the coordinate lines in VR

### PR DESCRIPTION
**Pull request by zingale**
## PR Checklist

- [ ] New features are documented, with docstrings and narrative docs
- [ ] Adds a test for any bugs fixed. Adds tests for new features.

### CODE REVIEW

**Comment by zingale** [comment]
grrr... my editor perturbed some source...

**Comment by zingale** [comment]
pre-commit.ci autofix

**Comment by zingale** [comment]
Here's the old (now default) rendering:

galaxy0030_vr_coords

and here's an alternate, using `sc.annotate_axes(alpha=0.01, thick=3)`

galaxy0030_vr_coords_new

**Review by neutrinoceros — COMMENTED** [review]
A couple minor comments:
- `thickness` might be more meaningful as a parameter name than `thick`
- I'd rather we don't allow instantiation of `CoordinateVectorSource` with all-positional arguments (i.e. `CoordinateVectorSource(None, 1.0, 1)`). In other words I suggest making `thick`/`thickness` a *keyword-only* argument.
- `CoordinateVectorSource`'s docstring needs to be updated

Otherwise that's a great accessibility feature, thank you !

**Comment by zingale** [comment]
thanks for the comments!  I've updated it now.

File: yt/visualization/volume_rendering/scene.py [hunk 1]
```diff
@@ -845,7 +844,7 @@ def annotate_axes(self, colors=None, alpha=1.0, thick=1):
             ``alpha`` is ignored.
         alpha : float, optional
             The opacity of the vectors.
-        thick : int, optional
+        thickness : int, optional
```
Comment (neutrinoceros) [inline yt/visualization/volume_rendering/scene.py:847]: does it have to be an int, or would a float work too ?

**Review by neutrinoceros — DISMISSED** [review]

**Comment by neutrinoceros** [comment]
pre-commit.ci autofix

**Pull request merged**